src/command/cloud/config.rs

```diff
@@ -65,7 +75,7 @@ impl Config {
         client: StudioClient,
         graph: &GraphRefOpt,
     ) -> RoverResult<RoverOutput> {
-        eprintln!("Fetching cloud config for: {}", graph.graph_ref);
+        println!("Fetching cloud router config for: {}", graph.graph_ref);
```

Comment:
We might have discussed this before but if these are log lines is there a reason they're not going to stderr?